src/main/java/uk/gov/hmcts/reform/orgrolemapping/data/BatchLastRunTimestampEntity.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.orgrolemapping.data;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.Entity;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Id;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+import java.io.Serializable;
+import java.io.Serial;
+import java.util.Date;
+
+@Builder(toBuilder = true)
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@SequenceGenerator(name = "batch_last_run_timestamp_id_seq", sequenceName = "batch_last_run_timestamp_id_seq",
+        allocationSize = 1)
+@Entity(name = "batch_last_run_timestamp")
+public class BatchLastRunTimestampEntity implements Serializable {
+
+    @Serial
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @Column(name = "id", updatable = false)
+    @GeneratedValue(strategy = GenerationType.IDENTITY, generator = "batch_last_run_timestamp_id_seq")
+    private String batchLastRunTimestampId;
+
+    //e.g. 2000-01-01T00:00:00
+    // JDBC and JPA 2.2 you are supposed to map it to java.time.OffsetDateTime
+    // (or to java.util.Date or java.sql.Timestamp of course)
+    @Column(name = "last_user_run_date_time", nullable = false)
+    @Temporal(TemporalType.DATE)
+    @UpdateTimestamp
+    private Date lastUserRunDatetime;
```

Comment:
```suggestion
    private LocalDateTime lastUserRunDatetime;
```

Reply:
the problem with `@UpdateTimestamp` is that any field annotated with it, gets automatically updated by hibernate to the current timestamp when there is a change to the entity. 

this is not the behaviour we want as we want to manually update the timestamp ourselves